pkg/cmd/dev/logic.go

```diff
@@ -93,7 +93,7 @@ func (d *dev) testlogic(cmd *cobra.Command, commandLine []string) error {
 			selectorPrefix = "TestLogic"
 		case "ccl":
 			testTarget = "//pkg/ccl/logictestccl:logictestccl_test"
-			selectorPrefix = "TestCCLLogic"
+			selectorPrefix = "Test(CCL|Tenant)Logic"
```

Comment:
Technically this is more tests than `make testccllogic` ran, right?

Reply:
This commit updates the Makefile too